### PR TITLE
Transaction Objects

### PR DESCRIPTION
**Pull request by keefertaylor**
Transform objects that are received from Conseil queries into transactions.

Addresses #2

### CODE REVIEW

File: TezosKit/Conseil/FetchSentTransactionsRPC.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ///   - apiKey: The API key to send in the request headers.
+  ///   - platform: The platform to query.
+  ///   - network: The network to query.
+  public init(account: String, limit: Int, apiKey: String, platform: ConseilPlatform, network: ConseilNetwork) {
+    let endpoint = "/v2/data/" + platform.rawValue + "/" + network.rawValue + "/operations"
+    let headers = [
+      Header.contentTypeApplicationJSON,
+      Header(field: "apiKey", value: apiKey)
+    ]
+    let payload = "{\"fields\": [\"timestamp\", \"source\", \"destination\", \"amount\", \"fee\"],\"predicates\": [{\"field\": \"kind\",\"set\": [\"transaction\"],\"operation\": \"eq\",\"inverse\": false}, {\"field\": \"source\",\"set\": [\"\(account)\"],\"operation\": \"eq\",\"inverse\": false}],\"orderBy\": [{\"field\": \"timestamp\",\"direction\": \"desc\"}],\"limit\": \(limit)}"
```
Comment (anonymoussprocket) [inline TezosKit/Conseil/FetchSentTransactionsRPC.swift]: operation_group_hash, block_hash might be useful to have as well

File: TezosKit/Conseil/Transaction.swift [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    public static let source = "source"
+    public static let destination = "destination"
+    public static let amount = "amount"
+    public static let fee = "fee"
+    public static let timestamp = "timestamp"
+  }
+
+  public let source: String
+  public let destination: String
+  public let amount: Tez
```
Comment (anonymoussprocket) [inline TezosKit/Conseil/Transaction.swift]: These values will be in µtz

File: TezosKit/Conseil/Transaction.swift [hunk 3]
```diff
@@ -0,0 +1,39 @@
+// Copyright Keefer Taylor, 2019
+
+import Foundation
+
+/// A transaction retrieved from Conseil.
+public struct Transaction {
```
Comment (anonymoussprocket) [inline TezosKit/Conseil/Transaction.swift]: This covers transactions reasonably well; you may want to add `Invocation` as a type as well and expose `parameters` there.

File: TezosKit/Conseil/FetchSentTransactionsRPC.swift (hunk 1 above)
Comment (jlaws) [inline TezosKit/Conseil/FetchSentTransactionsRPC.swift]: Use the JSON keys?

Comment (keefertaylor) [reply TezosKit/Conseil/FetchSentTransactionsRPC.swift]: I plan to create a query builder class in a future PR.

Comment (keefertaylor) [reply TezosKit/Conseil/FetchSentTransactionsRPC.swift]: Sorry, I hadn't properly merged master into this branch. Fields is now empty.

File: TezosKit/Conseil/Transaction.swift (hunk 2 above)
Comment (keefertaylor) [reply TezosKit/Conseil/Transaction.swift]: Thanks. Updated tests.

**Pull request merged**